datasetinsights/io/gcs.py

```diff
@@ -101,3 +109,52 @@ def download_file_from_gcs(cloud_path, local_path, filename, use_cache=True):
 
     # TODO(YC) Should run file checksum before return.
     return local_filepath
+
+
+def download_folder_from_gcs(cloud_path, local_path):
```

Comment:
I would expect the below code to be in GCSDownloader. Together with download_file_from_gcs to be in the GCSDownloader. The reason being the if the GCSClient would evolve to take a service account in the constructor this is now not possible as the functions in this module depends on the GCSClient but assumes it takes no arguments.

Reply:
+1